.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="11" project-jdk-type="JavaSDK">
```

Comment:
@tmicro do you know why this is changing?

Reply:
It's probably because Gavin has a different version of Android Studio.  Let's not merge this file, and add it to .gitignore.

Reply:
I still see this file in the changes